### DISCUSSION TITLE
Postgres support planned?

### DISCUSSION THREAD

**Discussion by romu70** [beromir/Servas#93] [Ideas]
All in the title I guess 😉

**Comment by beromir** [comment]
Hi,
not planned at the moment. But I'm curious, what are the benefits over MySQL or SQLite?

**Comment by romu70** [comment]
You can ;)

I self host a bunch of services on my synology nas, and all of them store their data in a Postgres DB. And I don't want:
- overload my server with several DB engine
- having to learn another DB engine

I began with MySQL, a long time ago, and observed that all services I was interested in moved to Postgres. So I did as well.

**Reply by beromir** [reply]
I see. Laravel supports PostgreSQL. So that should be theoretically possible. I will take a look, when I have time.
